science/gnss-sdr/Portfile

```diff
@@ -16,6 +17,8 @@ platforms           darwin
 
 dist_subdir         gnss-sdr
 
+compiler.blacklist-append {clang < 602}
```

Comment:
Please add a comment above this what it does. Also, can you add in the equivalent for GCC?

Reply:
Here I followed https://trac.macports.org/ticket/58516

There are other ports that [do that](https://github.com/macports/macports-ports/search?q=%22%7Bclang+%3C+602%7D%22&unscoped_q=%22%7Bclang+%3C+602%7D%22) without specifying the equivalent for GCC.

I will add a comment saying "Requires C++14"

Reply:
Fair enough. For better or worse, some of us are working on supporting back to 10.[45] PPC, which right now requires using GCC[567], which work surprisingly well overall replacing Appe's default GCC. There's also an effort to get Clang working on the older OSs, which if successful would be really great!

Reply:
GCC's isn't as clear, except that by GCC5 C++14 was pretty much all there < https://gcc.gnu.org/projects/cxx-status.html#cxx14 >.

Reply:
So, yes, please add the comment you propose & then block up to GCC49. Eventually we'll have better compiler selection support & all this crud can go away. Until then, to each ...

Reply:
Not sure about how to block up to gcc49. I'm trying this:

```
# Requires C++14
compiler.blacklist-append {clang < 602} \
    gcc-4.0 \
    apple-gcc-4.0 \
    gcc-4.2 apple-gcc-4.2 llvm-gcc-4.2 macports-llvm-gcc-4.2 \
    macports-gcc-4.3 \
    macports-gcc-4.4 \
    macports-gcc-4.5 \
    macports-gcc-4.6 \
    macports-gcc-4.7 \
    macports-gcc-4.8 \
    macports-dragonegg-*
```